setup.py

```diff
@@ -108,7 +105,6 @@
         "License :: OSI Approved :: Apache Software License",
         # supported python versions
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.5",
```

Comment:
we should add a `python_requires` https://stackoverflow.com/a/45362425/1906073

Reply:
(this will also make sure any python 3.5 installations won't update to a released package without python 3.5 support)